radio/adaptors/open_clip_adaptor.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+from argparse import Namespace
+
+import torch
+from torch import nn
+import torch.nn.functional as F
+
+from radio.radio_model import RADIOModel
+from radio.adaptors.adaptor_registry import adaptor_registry, dict_t, state_t
+from radio.adaptors.mlp import create_mlp_from_state
+
+
+class OpenCLIP_RADIO(nn.Module):
+    def __init__(self, base_model: RADIOModel, main_config: Namespace, adaptor_config: dict_t, state: state_t):
+        super().__init__()
+
+        self.base_model = base_model
+        self.head_mlp = create_mlp_from_state(main_config.mlp_version, state, 'head.')
+        self.feat_mlp = create_mlp_from_state(main_config.mlp_version, state, 'feature.')
```

Comment:
I suppose it can't hurt but what would CLIP features be used for?

Reply:
Not sure. But maybe users have built out machinery that uses those features for something.